### PR TITLE
DDP-7103 add i18n for study builder

### PR DESCRIPTION
**Pull request by aglushko-broad**
add i18n for study builder UI

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts [hunk 1]
```diff
@@ -69,41 +69,52 @@ export class Auth0AdapterService implements OnDestroy {
...
+                clientID: clientId,
+                studyGuid: this.configuration.studyGuid,
+                responseType: 'code',
+                audience,
+                scope: 'offline_access openid profile',
+                redirectUri: this.configuration.auth0CodeRedirect
+            });
+        } catch (e) {
+            console.error(e);
+            throw new Error(e);
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts]: Why was this necessary?

Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts]: it wasn't but while I was working on i18n I faced again that old error about absence of adminClientId. The system throws  just a strange message about http interceptors in the console and that's it. Remember how we spent almost all day (or two days?) for pancan in the past and you realized it's related to adminClientId absence eventually and we agreed it would be better to improve the error logging in the future. So I decided to do it.

I can do it in a separate PR

Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts]: created separate PR https://github.com/broadinstitute/ddp-angular/pull/966

**Review by alextanlan — APPROVED** [review]
